File: src/main/java/eterea/api/rest/model/Feriado.java
/**
 * 
 */
package eterea.api.rest.model;

import java.io.Serializable;
import java.time.OffsetDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Table
@Entity
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class Feriado extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = -721334245380889160L;

	@Id
	@Column(name = "fer_fecha")
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
	private OffsetDateTime fecha;

	@Column(name = "fer_nombre")
	private String nombre;

	@Column(name = "clave")
	private Long feriadoId;

}

File: src/main/java/eterea/core/api/rest/kotlin/model/Feriado.kt
package eterea.core.api.rest.kotlin.model

import com.fasterxml.jackson.annotation.JsonFormat
import jakarta.persistence.*
import java.time.OffsetDateTime

@Entity
data class Feriado(

    @Id
    @Column(name = "fer_fecha")
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    var fecha: OffsetDateTime? = null,

    @Column(name = "fer_nombre")
    var nombre: String? = null,

    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "clave")
    var feriadoId: Long? = null

) : Auditable()
